styles/instagram/catppuccin.user.css

```diff
@@ -657,7 +659,10 @@
     .xnz67gz {
       background-color: @catppuccin[@@lookup][@base];
     }
-
+    [style*="background-color: rgb(55, 151, 240);"] {
+      background-color: @catppuccin[@@lookup] [ @blue] !important;
+      color: @catppuccin[@@lookup] [ @mantle];
+    }
     .xvbhtw8 {
       background-color: @catppuccin[@@lookup][@mantle];
     }
```

Comment:
I'm confused why are these not just `@base` and `@blue` and `@mantle`?

Reply:
uh i have no idea why honestly, i think it's something to do with the fact that they're on a subdomain? `@base` or the others doesn't work, so i have to do it like this

Reply:
Its because you didn't paste all the variables in again from the template:
https://github.com/catppuccin/userstyles/blob/main/template/catppuccin.user.css#L22-L48

Reply:
> Its because you didn't paste all the variables in again from the template: https://github.com/catppuccin/userstyles/blob/main/template/catppuccin.user.css#L22-L48

ah ok